lib/moonshot/controller_config.rb

```diff
@@ -21,5 +25,17 @@ def initialize
       @plugins = []
       @show_all_stack_events = false
     end
+
+    def parameter_strategy=(value)
+      @parameter_strategy =
+        case value.to_sym
+        when :default
+          Moonshot::ParameterStrategy::DefaultStrategy.new
+        when :merge
+          Moonshot::ParameterStrategy::MergeStrategy.new
+        else
+          raise Thor::Error, "Unknown parameter strategy: #{value}"
```

Comment:
Assuming we keep this, we should raise a non-Thor specific error for the same reason, it's possible to use the controller without a Thor tool.